api/src/db/migrations/2024.06.07T00.36.14.clean-user-group-names.ts

```diff
@@ -1,12 +1,23 @@
 import type { Migration } from "@/db/umzug"
-import { UserGroup } from "@/models"
+import { QueryTypes } from "sequelize"
 
-export const up: Migration = async () => {
-  await UserGroup.findEach(async (userGroup) => {
-    await userGroup.update({
-      name: userGroup.name.trim().replace(/\s+/g, " "),
-    })
-  })
+export const up: Migration = async ({ context: queryInterface }) => {
+  const rows: { id: number; name: string }[] = await queryInterface.sequelize.query(
```

Comment:
FYI: you can also write this as
```suggestion
  const rows = await queryInterface.sequelize.query<{ id: number; name: string }>(
```